### PR TITLE
TFW-4253 - Add auto-scroll and new OR button

### PR DESCRIPTION
**Pull request by tVeratti**
Screen Shot 2019-08-01 at 3 43 13 PM

### CODE REVIEW

File: components/universalSearch/helperModal.beta.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@ $color-inactive: rgba($color-black, .5);
 
 .helper-modal {
     display: flex;
-    min-height: 20rem;
+    min-height: #{20rem - 5.7rem - 2.2rem};
```
Comment (jasonhale) [inline components/universalSearch/helperModal.beta.scss]: Why the math?  A comment would help here.  And are any of those needing to be vars?

File: components/universalSearch/helperModal.js [hunk 2]
```diff
@@ -74,12 +81,12 @@ export class HelperModalComponent extends PureComponent {
             }
         }
 
-        if (type !== 'autocomplete') {
+        if (type === 'autocomplete') {
```
Comment (jasonhale) [inline components/universalSearch/helperModal.js]: 💯

File: components/universalSearch/helperModal.beta.scss (hunk 1 above)
Comment (tVeratti) [reply components/universalSearch/helperModal.beta.scss]: The minimum height of the modal was meant to be 200 from top of the header to the bottom, not just the content area. I subtracted the size of the header and the surrounding margins from the modal component itself. I can definitely explain that in a comment.

Comment (jasonhale) [reply components/universalSearch/helperModal.beta.scss]: Yeah, that'd really help.  We have no reference to those measurements in case something changes.

Comment (jasonhale) [reply components/universalSearch/helperModal.beta.scss]: yeah, that'd help.  or just set it to the right value.  as it is is a little confusing as to where those come from.

**Pull request merged**